Trackify/Trackify/Flight.swift

```diff
@@ -10,14 +10,27 @@ import Foundation
 import AWSDynamoDB
 
 class Flight : AWSDynamoDBObjectModel, AWSDynamoDBModeling {
+    fileprivate var df = DateFormatter()
+    fileprivate var timeString: String?
+    fileprivate var dateString: String?
+    fileprivate var date: Date?
+    
     var airline: String?
     var flightNumber: String?
-    var date: Date?
     var departureAirport: String?
     var destinationAirport: String?
     var confirmation: String?
-    var email:String?
-    var datetime:String?
+    var email: String?
+    var datetime: String? {
+        didSet {
+            df.dateFormat = "MM-dd-yyyy HH:mm"
+            self.date = df.date(from: datetime!)
+            df.dateFormat = "h:mm a"
+            self.timeString = df.string(from: self.date!)
+            df.dateFormat = "MMMM d, yyyy"
+            self.dateString = df.string(from: self.date!)
```

Comment:
Q: why is the df.dateformat changed to "h:mm a" as opposed to "HH:mm"? similar question with "MMMM d, yyyy". I'm assuming just the syntax for dateformat but just seems strange to me.